requirements-aws.txt

```diff
@@ -1,3 +1 @@
-boto3==1.4.4
 django-smtp-ssl==1.0
```

Comment:
Yes, we might add more in the future. Also, we should split up `requirements.txt` more.